getting-started/other-frameworks/vitest/your-first-vitest-run.md

```
@@ -0,0 +1,131 @@
+Add the converted reports directory and the JUnit XML file to your .gitignore to avoid pushing temporary generated reports to your repository.
+
+```
+.currents
+results.xml
+```
+
+</details>
+
+## Your First Vitest Run
+
+#### Step 1: Run the tests
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)